src/pages/instructor/components/activity-view.tsx

```diff
@@ -25,85 +22,69 @@ export function ActivityView(props: {
 
   if (!selectedDocId) {
     return (
-      <EducationalProvider
-        initialActivityId={activityId}
-        initialAssignmentId={assignmentId}
-        initialCourseId={courseId}
-        initialSectionId={sectionId}
-      >
-        <ViewUserGoogleDocs
-          goToDoc={(docId: string) => {
-            setSelectedDocId(docId);
-          }}
-          onHistoryClicked={() => {
-            console.log('history clicked');
-          }}
-        />
-      </EducationalProvider>
+      <ViewUserGoogleDocs
+        goToDoc={(docId: string) => {
+          setSelectedDocId(docId);
+        }}
+        onHistoryClicked={() => {
+          console.log('history clicked');
```

Comment:
will this function be implemented in the future? otherwise, you can remove the console.log

Reply:
yes, in the next pr